lib/commands/kendoui-install.ts

```diff
@@ -0,0 +1,106 @@
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let packages = this.getKendoPackages().wait();
+
+			let downloadUri: string;
+			if(this.$options.latest) {
+				let latestPackage = _.first(packages);
+				let sameDateItems = _.filter(packages, pack => pack.Version === latestPackage.Version);
+				if(sameDateItems.length > 1) {
+					downloadUri = this.selectKendoVersion(sameDateItems).wait();
+				} else {
+					downloadUri = latestPackage.DownloadUrl;
+				}
+			} else {
+				downloadUri = this.selectKendoVersion(packages).wait();
+			}
+
+			let confirm = this.$prompter.confirm(
+				"This operation will overwrite existing Kendo UI framework files and " +
+				"any changes will be lost. ".red.bold +
+				"Are you sure you want to continue?",
+				() => true).wait();
+			if (!confirm) {
+				return;
+			}
+
+			this.updateKendoFiles(downloadUri).wait();
+
+		}).future<void>()();
+	}
+
+	private selectKendoVersion(packages: Server.IKendoDownloadablePackageData[]): IFuture<string> {
+		return ((): string => {
+			this.$logger.out("You can download and install the following Kendo UI packages.");
+			this.$logger.out(this.$kendoUIService.getKendoPackagesAsTable(packages));
+			let schema: IPromptSchema = {
+				type: "input",
+				name: "packageIdx",
+				message: "Enter the index of the package that you want to install",
+				validate: (value: string) => {
+					let num = parseInt(value, 10);
+					return !isNaN(num) && num >= 1 && num <= packages.length ? true : "Valid values are between 1 and " + packages.length;
+				}
+			};
+
+			let choice = this.$prompter.get([schema]).wait();
+			let packageIdx = parseInt(choice.packageIdx, 10) - 1;
+			let selectedPackage = packages[packageIdx];
+			if (selectedPackage.HasReleaseNotes) {
+				let shouldShowReleaseNotes = this.$prompter.confirm(
+				"Do you want to review this package's release notes?",
+				() => true).wait();
+				if (shouldShowReleaseNotes) {
+					this.$opener.open(selectedPackage.ReleaseNotesUrl);
+				}
+			}
+
+			this.$logger.trace("Selected package is:");
+			this.$logger.trace(selectedPackage);
+			return selectedPackage.DownloadUrl;
+		}).future<string>()();
+	}
+
+	private updateKendoFiles(downloadUri: string): IFuture<void> {
+		return (() => {
+			temp.track();
+
+			let filepath = temp.path({suffix: ".zip", prefix: "abkendoupdate-"});
+			let file = this.$fs.createWriteStream(filepath);
+			let fileEnd = this.$fs.futureFromEvent(file, "finish");
+			this.$httpClient.httpRequest({ url: downloadUri, pipeTo: file }).wait();
+			fileEnd.wait();
+
+			let outDir = path.join(this.$project.getProjectDir().wait(), "kendo");
```

Comment:
maybe it is worth backing up current kendo dir and in case of unzip failure (or something else happens), to return the previous state of the project.